dotnet/src/Connectors/Connectors.AzureOpenAI/Settings/AzureOpenAIPromptExecutionSettings.cs

```diff
@@ -101,8 +101,8 @@ public static AzureOpenAIPromptExecutionSettings FromExecutionSettingsWithData(P
     }
 
     #region private ================================================================================
-
-    private AzureChatDataSource? _azureChatDataSource;
+    [Experimental("SKEXP0010")]
+    private AzureSearchChatDataSource? _azureChatDataSource;
```

Comment:
Is `Experimental` flag required on private variable?

Reply:
If I don't do that I need to add a pragma for the Azure Experimental in here.